openfisca_uk/variables/finance/income.py

```diff
@@ -241,3 +241,58 @@ class base_net_income(Variable):
     entity = Person
     label = "Existing net income for the person to use as a base in microsimulation"
     definition_period = YEAR
+
+
+class is_apprentice(Variable):
```

Comment:
Should this have a false default, or is false the default default?

Reply:
Technically zero is the default default, but it'll be cast to False. But I think we should specify in the abscence of a formula